openmetadata-ui/src/main/resources/ui/src/styles/x-master.css

```diff
@@ -257,6 +257,20 @@ pre {
   background-color: #e8e7e4;
 }
 
+/* google sso layout style  */
+.googlecontainer {
```

Comment:
Can we move this class to `app.less` file?

Reply:
@Sachin-chaurasiya  updated